glide.yaml

```diff
@@ -1,5 +1,7 @@
 package: github.com/giantswarm/kvm-operator
 import:
+- package: github.com/giantswarm/apiextensions
+  version: clientset
```

Comment:
We need to pin this branch until the migration is done. I will make sure we get back to master after we run on CRDs only.